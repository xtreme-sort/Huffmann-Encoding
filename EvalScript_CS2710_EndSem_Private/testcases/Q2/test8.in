DeriveHuffmanTree
30
1 00111100
3 11111
8 11011
B 0011101
E 1000
H 101111
I 0010
J 00111101
L 0000
O 10010
R 0011111
T 111101
U 110100
W 11000
X 0001
Y 0101
Z 111100
a 110101
b 0110
h 11100
i 101110
j 00110
l 10110
n 11001
o 0111
q 11101
r 0100
s 1010
y 10011
z 0011100
Quit
