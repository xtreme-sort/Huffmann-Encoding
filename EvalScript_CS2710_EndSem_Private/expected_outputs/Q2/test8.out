((((L,X),(I,(j,((z,B),((1,J),R))))),((r,Y),(b,o))),(((E,(O,y)),(s,(l,(i,H)))),(((W,n),((U,a),8)),((h,q),((Z,T),3)))));
